From c1b07082dea158d5a3a3c1f1f5ad719db16db0a4 Mon Sep 17 00:00:00 2001
From: Philip Chimento <pchimento@igalia.com>
Date: Mon, 4 Sep 2023 13:03:15 -0700
Subject: [PATCH] Comment out dependencies

Skip installing the dependencies only needed for ChromeOS since we are
not building ChromeOS.

Note: this patch is applied in setup.sh rather than patch.sh.

---

diff --git a/build/install-build-deps.py b/build/install-build-deps.py
index 9d927e632..2bd1def2a 100755
--- a/build/install-build-deps.py
+++ b/build/install-build-deps.py
@@ -259,13 +259,13 @@ def dev_list():
       "zip",
   ]
 
-  # Packages needed for chromeos only
-  packages += [
-      "libbluetooth-dev",
-      "libxkbcommon-dev",
-      "mesa-common-dev",
-      "zstd",
-  ]
+  # # Packages needed for chromeos only
+  # packages += [
+  #     "libbluetooth-dev",
+  #     "libxkbcommon-dev",
+  #     "mesa-common-dev",
+  #     "zstd",
+  # ]
 
   if package_exists("realpath"):
     packages.append("realpath")
@@ -378,11 +378,11 @@ def lib_list():
       "zlib1g",
   ]
 
-  # Run-time libraries required by chromeos only
-  packages += [
-      "libpulse0",
-      "libbz2-1.0",
-  ]
+  # # Run-time libraries required by chromeos only
+  # packages += [
+  #     "libpulse0",
+  #     "libbz2-1.0",
+  # ]
 
   if package_exists("libffi8"):
     packages.append("libffi8")
-- 
2.41.0

